backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/datamodels/DataCatalogueCohortStagingLoader3.java

```diff
@@ -1,35 +1,32 @@
 package org.molgenis.emx2.datamodels;
 
-import static org.molgenis.emx2.datamodels.DataCatalogueLoader.createSchema;
 import static org.molgenis.emx2.datamodels.DataCatalogueNetworkStagingLoader.SHARED_STAGING;
 
 import org.molgenis.emx2.Database;
 import org.molgenis.emx2.Schema;
 
-public class DataCatalogueCohortStagingLoader3 implements AvailableDataModels.DataModelLoader {
+public class DataCatalogueCohortStagingLoader3 extends AbstractDataLoader {
 
   static String DATA_CATALOGUE = "DataCatalogue";
 
   @Override
-  public void load(Schema schema, boolean includeDemoData) {
+  void loadInternalImplementation(Schema schema, boolean includeDemoData) {
     // create shared schemas
-    createDataCatalogue3(schema.getDatabase());
+    createSharedStaging(schema.getDatabase());
     // create the schema
     createSchema(schema, "datacatalogue3/stagingCohorts/molgenis.csv");
   }
 
-  static void createDataCatalogue3(Database db) {
+  static void createSharedStaging(Database db) {
```

Comment:
keep the '3' in the method name or make it private , otherwise there is a change a dev call the wrong method by accident

Reply:
good suggestion